Backend/routes/userRoutes.js

```diff
@@ -441,9 +441,9 @@ router.post('/songs', authenticateJWT, async (req, res) => {
                 title,
                 artist,
                 album,
-                genre,
-                mood,
-                tempo,
+                genre: genre.toLowerCase(),
+                mood: mood.toLowerCase(),
+                tempo: mood.toLowerCase(),
```

Comment:
this is the storage for the user's tag right? this might be a little nit picky but I think it's cleaner to store the raw data as is and filter it later when you are manipulating it, just a suggestion tho

Reply:
Got it, I'll fix that